client/client.go

```diff
@@ -16,6 +18,24 @@ import (
 	"golang.org/x/crypto/ssh/agent"
 )
 
+// InstallPublicFiles installs the public part of the cert and key.
+func InstallPublicFiles(prefix string, cert *ssh.Certificate, pub ssh.PublicKey) error {
+	if prefix == "" {
+		return nil
+	}
+	pubKey := prefix + ".pub"
+	pubCert := prefix + "-cert.pub"
```

Comment:
Ah, yes. That was a lot longer when I was passing in the full config.

The same is true for `pubTxt` and `certPubTxt` but the WriteFile lines become kind of dense at that point. The compiler is doing the same thing either way so I'd prefer keeping that to make it a bit easier to read.